decimal64.go

```diff
@@ -99,19 +99,19 @@ func (dec *decParts) separation(eDec decParts) int {
 
 // removeZeros removes zeros and increments the exponent to match.
 func (dec *decParts) removeZeros() {
-	zeros := countTrailingZeros(dec.significand)
-	dec.significand /= powersOf10[zeros]
+	zeros := countTrailingZeros(dec.significand.lo)
+	dec.significand.lo /= powersOf10[zeros]
```

Comment:
it seems like `significand` is refactored to `significand.lo` all over the file, what is the change referencing to?

Reply:
significand got changed from a uint64 to a uint128T, 
significand.lo is of the same type as the previous significand
after calculation is done, significand.hi gets divided into significand.lo